va/va_vpp.h

```diff
@@ -439,6 +439,8 @@ typedef enum _VAProcHighDynamicRangeMetadataType {
     VAProcHighDynamicRangeMetadataNone = 0,
     /** \brief Metadata type for HDR10. */
     VAProcHighDynamicRangeMetadataHDR10,
+    /** \brief Metadata type for HDR10+. */
+    VAProcHighDynamicRangeMetadataHDR10Plus,
     /** \brief Number of Metadata type. */
     VAProcHighDynamicRangeMetadataTypeCount
```

Comment:
It seems there is an API/ABI change, VAProcHighDynamicRangeMetadataTypeCount will be changed. Maybe we should deprecate VAProcHighDynamicRangeMetadataTypeCount in the future.